src/handler/newOrder.ts

```diff
@@ -6,6 +6,7 @@ import { ValidationError } from './errors'
 import { addQuoteIdPrefix, constructQuoteResponse, preprocessQuote } from '../quoting'
 
 import { assetDataUtils, SignedOrder } from '0x-v2-order-utils'
+import { buildSignedOrder as buildRFQV1SignedOrder } from '../signer/rfqv1'
 import { buildSignedOrder } from '../signer/pmmv5'
 import { buildSignedOrder as buildLagacyOrder } from '../signer/pmmv4'
```

Comment:
please remove the pmmv4 order related code, it's been deprecated.

Reply:
Fixed in d06dba6